react-frontend/src/model/SwimlaneData.js

```diff
@@ -0,0 +1,9 @@
+export default class SwimlaneData {
```

Comment:
yup, it's better to have everything by 'name' instead by object :) - at least I know what can I expect inside of such object, with raw 'maps' I can only guess what's next